### ISSUE TITLE
Cross-Origin Request Blocked

### ISSUE THREAD

**Issue by alko89** [OriginTrail/AAP-market-simulator#7]
I'm unable to start the backend from the frontend.

When I click start I get CORS error:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:3001/start. (Reason: CORS request did not succeed).
```

**Comment by kotlarmilos** [comment]
Could you please provide the following:
- Logs from the browser console
- OS details
- If you have any specific settings of CORS policy

**Comment by alko89** [comment]
There are a bunch of logs when I open the page, but when I click the button, only the course error is thrown, so I'm not sure if its related.
![image](https://user-images.githubusercontent.com/2822030/96369332-9127d380-1159-11eb-8739-ebce665828f4.png)

I'm using Arch Linux and Firefox 81.0.2.

I don't have any CORS settings set.

**Comment by kotlarmilos** [comment]
Thank you for providing the logs.
Could you try to open the simulator in any other browser (i.e. chrome)?

We have enabled CORS requests in nodejs express.

**Comment by bottomshorter** [comment]
![image](https://user-images.githubusercontent.com/47401992/96490348-89b60800-11f5-11eb-9d4d-e7ae6c802672.png)

i get some similar but also additional errors on chrome. It says the DOM property is 'readonly' but its asking if it means 'readOnly'.

**Comment by alko89** [comment]
Actually I figured out that the backend didn't even open a port (should be 3001 right?) and I also have an issue that I have to kill it with `SIGKILL` to stop it. It seems like its frozen.
I don't get any console output and I thought its fine at first.
Do I have to wait some time to fully initialize?
```
% netstat -tulpn | grep node
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
tcp        0      0 0.0.0.0:3000            0.0.0.0:*               LISTEN      73610/node
```

**Comment by alko89** [comment]
I figured it out, it seems the instructions are a bit incorrect. My config path was set to `export CONFIG_PATH=nodejs/config/config.json` but should be `export CONFIG_PATH=config/config.json` and so the program looped in `changeFileAttributeAtomic` because it wasn't able to lock the file (didn't exist on that path).

Then I had to fix the path to the worker script to `modules/simulator/simulator-worker.js` and was able to get it working.

It seems the instructions are intended to run the node from the project root directory, not `nodejs`. :)

I can make a PR with the fix.
